File: dart/28-05-2024.dart
import 'dart:io';

void main(){

  print('ingrese numero de estudiantes');

  int? n = int.parse(stdin.readLineSync()!);

  for ( var numero = 1; numero <= n; numero++){

    print ('ingrese nota');
    int? n1 = int.parse(stdin.readLineSync()!);

    print ('ingrese nota');
    int? n2 = int.parse(stdin.readLineSync()!);

    print ('ingrese nota');
    int? n3 = int.parse(stdin.readLineSync()!);

    double nota = n1 + n2 + n3 / 3;

    if (nota < 6){
      print ('REPROBADO');

    }else{ 
      print ('APROBADO');

    }


  }
}

File: ruby/28-05-2024.rb
puts "ingrese numero de estudiantes"
    n = gets.chomp.to_f

for numero in 1..n

    puts "ingrese nota"
    n1 = gets.chomp.to_f

    puts "ingrese nota"
    n2 = gets.chomp.to_f

    puts "ingrese nota"
    n3 = gets.chomp.to_f

    nota = (n1 + n2 + n3)/ 3
    if nota < 6
       puts "REPOBADO"

    else
       puts "APROBADO"



    end


end    